sampleWorkspace/httpSchema/apollo.config.ts

```diff
@@ -3,6 +3,8 @@ export default {
     service: {
       name: "httpSchema",
       url: "http://localhost:7096/graphql",
+      // url: "https://localhost:7097/graphql",
+      // skipSSLValidation: true,
```

Comment:
This can be swapped in manually now to test the `skipSSLValidation` with a self-signed certificate - I'm not making an automated test for this right now, though.